docs/admin/serving/feature-flags.md

```diff
@@ -238,36 +240,33 @@ spec:
                   fieldPath: spec.nodeName
 ```
 
-## Kubernetes Dry Run
+## Kubernetes dry run
 
-* **Type**: extension
+* **Type**: Extension
 * **ConfigMap key:** `kubernetes.podspec-dryrun`
 
-This flag controls whether Knative will try to validate the Pod spec derived from the Knative Service spec using the Kubernetes API server before accepting the object.
+This flag controls whether Knative attempts to validate the Pod spec derived from a Knative Service spec by using the Kubernetes API server before accepting the object.
```

Comment:
```suggestion
This flag controls whether Knative attempts to validate the Pod spec derived from a Knative Service spec, by using the Kubernetes API server before accepting the object.
```

Reply:
It needn't be a comma right there specifically -- we just need something to break up the sentence.